pull-data.mjs

```diff
@@ -325,7 +355,9 @@ export default async function run({ github, context, dryRun = false }) {
       build_number: tag_name,
       prerelease: release.prerelease,
       created_at: release.created_at,
+      updated_at: release.created_at,
```

Comment:
is this intended to be always the same as created_at? it looks like it's never changed, so could be dropped